user_management/ui/views.py

```diff
@@ -33,6 +34,7 @@ def dispatch(self, request, *args, **kwargs):
         return super(VerifyUserEmailView, self).dispatch(request, *args, **kwargs)
 
     def get(self, request, *args, **kwargs):
+        next = resolve_url(settings.LOGIN_REDIRECT_URL)
```

Comment:
Should this be `self.request.GET.get('next', None) or resolve_url(settings.LOGIN_REDIRECT_URL)` so that the `settings.LOGIN_REDIRECT_URL` is a fallback as specified in https://docs.djangoproject.com/en/1.10/ref/settings/#login-redirect-url